spec/requests/story_items_spec.rb

```diff
@@ -0,0 +1,79 @@
+require 'spec_helper'
+
+RSpec.describe 'Stories Items', type: :request do
+  let(:admin) { create(:admin_user) }
+  let(:story) { create(:story) }
+
+  describe 'index' do
+    context 'when requesting without a user_key' do
+      before { get "/v3/stories/#{story.id}/items.json?api_key=#{admin.authentication_token}" }
+
+      it 'returns an error' do
+        response_attributes = JSON.parse(response.body)
+
+        expect(response_attributes).to eq ({ 'errors' => 'Mandatory parameter user_key missing' })
+      end
+    end
+
+    context 'when requesting with a user_key' do
+      before { get "/v3/stories/#{story.id}/items.json?api_key=#{admin.authentication_token}&user_key=#{story.user.api_key}" }
+
+      it 'returns story items for the user key' do
+        response_attributes = JSON.parse(response.body)
+  
+        expect(response_attributes).to eq (
+          story.set_items.sort_by(&:position).map do |item|
+            {
+              'record_id' => item.record_id,
```

Comment:
this can be work with new syntax

`'record_id`: item.record_id`.

But up to you.

Reply:
We are testing the API response here and we are parsing JSON. So this is appropriate. It will be just an extra step in the test to symbolize keys.